docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/GamWithOptionsAdvanced.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+using Microsoft.ML.Trainers.FastTree;
+
+namespace Samples.Dynamic.Trainers.BinaryClassification
+{
+    public static class GamWithOptionsAdvanced
+    {
+        // This example requires installation of additional NuGet package
```

Comment:
>installation [](start = 33, length = 12)

No tt?

Reply:
No.

---
In reply to: [275455757](https://github.com/dotnet/machinelearning/pull/3319#discussion_r275455757) [](ancestors = 275455757)

Reply:
no tt -qq

---
In reply to: [275571580](https://github.com/dotnet/machinelearning/pull/3319#discussion_r275571580) [](ancestors = 275571580,275455757)

Reply:
QQ

---
In reply to: [275581911](https://github.com/dotnet/machinelearning/pull/3319#discussion_r275581911) [](ancestors = 275581911,275571580,275455757)